src/registrar/templates/includes/summary_item.html

```diff
@@ -142,6 +145,9 @@ <h4 class="margin-bottom-05">Invited domain managers</h4>
         </ul>
       {% endif %}
     
+    <!-- this is just to prevent the portfolio from appearing on Org Overview page -->
+    {% elif sub_header_text == "Organization Admins" %}
+
     {% else %}
```

Comment:
```suggestion
    {% elif sub_header_text != "Organization admins"%}
```
(sentence-cased given the value passed in as sub_header_text is updated)